packages/common/vbmSignupDeadline.ts

```diff
@@ -0,0 +1,67 @@
+import { ImplementedState } from './stateInfo'
+import { State } from './states'
+
+const year = 2020
+const deadline = (month: number, day: number) => new Date(year, month-1, day)
+
+type StateDeadline =
+  Record<ImplementedState, Date | null>
+  & Partial<Record<State, Date | null>>
+
+export const vbmSignupDeadline: StateDeadline = {
+  Alabama: null,
```

Comment:
Why not omit the entires that are null?  Also, don't export this table.  People should use the function.

Reply:
The only reason why I didn't omit `null` deadlines is because type `StateDeadline` makes it mandatory for `ImplementedState` to be present on the `Record`, should I change this?

Reply:
But `Alabama` is not an `ImplementedState`, so we don't need to set it to null.  Sorry, I meant to remove the ones for not `ImplementedState`